util/file.py

```diff
@@ -473,3 +494,36 @@ def line_count(infname):
 def touch(fname, times=None):
     with open(fname, 'a'):
         os.utime(fname, times)
+
+def touch_empty(fname):
+    '''Make each file in fnames a zero-length file with the current timestamp.'''
+    with open(fname, 'w'):
+        pass
+
+def dump_file( fname, value ):
+    """store string in file"""
+    with open( fname, 'w' )  as out: 
+        out.write( str( value ) )
+
+def slurp_file( fname, noErr=False ):
+    """Read entire file into one string."""
+    if not fname or not isinstance(fname,str) or not os.path.isfile( fname ): 
```

Comment:
Maybe frame this as a try/except, with a `return None` in the exception handler if `noErr=True`